backend/de.metas.material/cockpit/src/main/java/de/metas/material/cockpit/availableforsales/AvailableForSalesQuery.java

```diff
@@ -33,7 +34,7 @@
 @Builder
 public class AvailableForSalesQuery
 {
-	int productId;
+	ProductId productId;
```

Comment:
pls annotate with `@NonNull`, so we don't have to worry about NPEs down the road